src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -81,4 +81,59 @@ public void testGetLastTradedPriceEmpty() throws Exception {
         .andExpect(jsonPath("$", is(Collections.emptyList())))
         .andReturn();
   }
+
+  @Test
+  public void testHistoricalPrices() throws  Exception {
+    MvcResult result = this.mvc.perform(
+            org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                .get("/iex/historicalPrices?symbol=FB&range=1m&date=20200409")
+                .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$[0].high").value(new BigDecimal("175.71")))
+        .andExpect(jsonPath("$[1].low").value(new BigDecimal("174.365")))
+        .andExpect(jsonPath("$[2].open").value(new BigDecimal("174.8")))
+        .andExpect(jsonPath("$[3].close").value(new BigDecimal("174.55")))
+        .andExpect(jsonPath("$[5].volume").value(new BigDecimal(1155)))
+        .andReturn();
+  }
+
+  @Test
+  public void testHistoricalPricesNoDate() throws  Exception {
+    MvcResult result = this.mvc.perform(
+        org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+            .get("/iex/historicalPrices?symbol=FB&range=1m")
+            .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$[0].high").value(new BigDecimal("175.71")))
+        .andExpect(jsonPath("$[1].low").value(new BigDecimal("174.365")))
+        .andExpect(jsonPath("$[2].open").value(new BigDecimal("174.8")))
+        .andExpect(jsonPath("$[3].close").value(new BigDecimal("174.55")))
+        .andExpect(jsonPath("$[5].volume").value(new BigDecimal(1155)))
+        .andReturn();
+  }
+
+  @Test
+  public void testHistoricalPricesNoRange() throws  Exception {
+    MvcResult result = this.mvc.perform(
+            org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+                .get("/iex/historicalPrices?symbol=FB&date=20200409")
+                .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isOk())
+        .andExpect(jsonPath("$[0].high").value(new BigDecimal("175.71")))
+        .andExpect(jsonPath("$[1].low").value(new BigDecimal("174.365")))
+        .andExpect(jsonPath("$[2].open").value(new BigDecimal("174.8")))
+        .andExpect(jsonPath("$[3].close").value(new BigDecimal("174.55")))
+        .andExpect(jsonPath("$[5].volume").value(new BigDecimal(1155)))
+        .andReturn();
+  }
+
+  @Test
+  public void testHistoricalPricesNoSymbol() throws Exception {
+    MvcResult result = this.mvc.perform(
+        org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+            .get("/iex/historicalPrices?symbol=")
+            .accept(MediaType.APPLICATION_JSON_VALUE))
+        .andExpect(status().isNotFound())
+        .andReturn();
+  }
```

Comment:
Still missing the following combinations:
* Range and Date provided
* Date and Symbol provided
* Only symbol provided
* Only range provided
* Invalid format range provided
* Invalid format date provided
* Symbol provided that doesn't exist

And the error scenario:
* IEX returns 500/server not available type error